### ISSUE TITLE
Multiple Drop Down Menus

### ISSUE THREAD

**Issue by mwloewen** [StraboSpot/StraboSpot2#589]
**Is your feature request related to a problem? Please describe.**
This MAY be specific to the Tephra Module in Testing: the form has multiple drop down menus. If you don't close the previous one and open another, you can have two open at once which makes page navigation difficult.

**Describe the solution you'd like**
When a new drop down is selected, automatically close previously open ones.

**Comment by jessicagood** [comment]
Yes, this is how it is for every form. But the drop down should automatically close when when you select one of the options so you shouldn't be left with open menus unless you don't select anything. Is this what is doing for you?
